tests/affine-local/e2e/all-page.spec.ts

```diff
@@ -85,7 +85,8 @@ test('allow creation of filters by favorite', async ({ page }) => {
   );
 });
 
-test('allow creation of filters by created time', async ({ page }) => {
+// new onboarding create pages that are all today
+test.skip('allow creation of filters by created time', async ({ page }) => {
```

Comment:
continue to skip or code forget to delete?

Reply:
🤔 yeah, maybe I can just remove them